package.json

```
@@ -1,6 +1,6 @@
 {
   "engines": {
-    "node": "12.15.0"
+    "node": "12.18.4"
```

```suggestion
    "node": "12.x"
```
We don't really care about the minor version